charts/ota-kafka-service/.helmignore

```diff
@@ -0,0 +1,21 @@
+# Patterns to ignore when building packages.
```

Comment:
Can remove the ota-kafka-service chart. Only the ota-kafka chart is required.

Reply:
Done.

Reply:
Done.